Gulpfile.js

```diff
@@ -253,8 +254,32 @@ gulp.task('images', ['clean'], function () {
         .pipe(gulp.dest('lib'));
 });
 
+gulp.task('greenkeeper', function () {
+    var packageJsonFilePath = path.join(__dirname, './package.json');
```

Comment:
No need for this var. Just call `readFile('package.json')` since we always run from the gulp in the root of the project